### DISCUSSION TITLE
How to clean cache?

### DISCUSSION THREAD

**Discussion by vdegenne** [saghen/blink.cmp#467] [Q&A] [answered]
What's the easiest way to clean cache? I tried to locate the cache file in `~/.local/state/.` but I can't find it.

**Comment by saghen** [comment, marked as answer]
`rm -r ~/.local/state/nvim/blink/cmp/fuzzy.db` should wipe the frecency data

**Reply by vdegenne** [reply]
Thanks @Saghen

**Reply by vdegenne** [reply]
btw it was `~/.local/share/` not `~/.local/state`
